servicemesh-install/topics/install.adoc

```diff
@@ -244,6 +233,12 @@ See the OpenShift documentation on https://docs.openshift.com/container-platform
 
 === Istio gateway example
 
+[WARNING]
+====
+Automatic route creation does not currently work with multi-tenancy. Set `ior_enabled` to `false` for multi-tenant installations.
+====
```

Comment:
Should we add this to the multitenant ServiceMeshMemberRoll example?

Reply:
This was a bullet under the "Known Issues" section in the multi-tenant documentation. I elevated it to a warning before the list of known issues.